client/my-sites/github-deployments/components/deployments-survey/utils.ts

```diff
@@ -0,0 +1,29 @@
+import cookie from 'cookie';
+
+const setSurveyCookie = ( name: string, value: string, maxAge: number ) => {
+	document.cookie = cookie.serialize( `github-deployments-${ name }`, value, {
+		path: '/',
+		maxAge,
+	} );
+};
+
+const getSurveyCookie = ( name: string ) => {
+	return cookie.parse( document.cookie )[ `github-deployments-${ name }` ];
+};
+
+export const maybeSetDeploymentDone = () => {
+	const cookieValue = getSurveyCookie( 'has-deployments' );
+	if ( cookieValue ) {
+		return;
+	}
+	setSurveyCookie( 'has-deployments', 'true', 365 * 24 * 60 * 60 ) /* 1 year */;
+	setSurveyCookie( 'hide-survey', 'true', 7 * 24 * 60 * 60 ) /* 7 days */;
```

Comment:
In what cases `has-deployments` and `hide-survey` won't exist together? Meaning, are we calling `hide-survey` somewhere else? Doesn't seem like it. I never see the survey after creating my first deployment. What's the intent here?

Reply:
The hide survey is a cookie that expires in 7 days, at which point it will be deleted and the survey will show. If u manually delete it and refresh the page you will see the survey.